File: 1-Iniciante/1021-Notas_e_Moedas-6/1021.py
x=float(input())

notas=(100.00, 50.00, 20.00, 10.00, 5.00, 2.00, 1.00, 0.50, 0.25, 0.10, 0.05, 0.01)
print("NOTAS:")
comp="nota"
for i in notas:
    if i == 1.00:
        print("MOEDAS:")
        comp="moeda"
    if x >= i:
        print(f"{int(x//i)} {comp}(s) de R$ {i:.2f}")
        x=x-i*(x//i)+0.00001
    else:
        print(f"0 {comp}(s) de R$ {i:.2f}")

File: 1-Iniciante/1021-Notas_e_Moedas-6/1021.cs
using System;

public class HelloWorld
{
    public static void Main(string[] args)
    {
        double[] notas = { 100.0,50.0, 20.0, 10.0, 5.0, 2.0 };
        double[] moedas = { 1.0, 0.50, 0.25, 0.10, 0.05, 0.01 };
        double n = double.Parse(Console.ReadLine());
        Console.WriteLine("NOTAS:");
        foreach (double note in notas)
        {
            if (n >= 0 && n <= 1000000.00)
            {
                int division = (int)(n / note);
                n = n % note;
                Console.WriteLine($"{division} nota(s) de R$ {note.ToString("F2")}");
                n = Math.Round( n, 2 );
            }
        }
        Console.WriteLine("MOEDAS:");
        foreach (double i in moedas)
        {
            if (n >= 0 && n <= 1000000.00)
            {
                int division = (int)(n / i);
                n = n % i;
                Console.WriteLine($"{division} moeda(s) de R$ {i.ToString("F2")}");
                n = Math.Round(n, 2);
            }
        }   
    }
}